toqito/channels/tests/test_bitflip.py

```diff
@@ -0,0 +1,80 @@
+"""Test cases for the bit-flip channel in Toqito."""
+
+import unittest

```

Comment:
We should use `pytest` (not `unittest`) to conform with how testing is done elsewhere in `toqito` for consistency.